packages/auth/src/accessToken/routes.ts

```diff
@@ -35,19 +37,28 @@ async function introspectToken(
   deps: ServiceDependencies,
   ctx: AppContext
 ): Promise<void> {
-  // TODO: request validation
   const { body } = ctx.request
-  if (body['access_token']) {
-    const introspectionResult = await deps.accessTokenService.introspect(
-      body['access_token']
-    )
-    if (introspectionResult) {
-      ctx.body = introspectionToBody(introspectionResult)
-    } else {
-      return ctx.throw(404, 'token not found')
+  if (!body['access_token']) {
+    ctx.status = 400
+    ctx.body = {
+      error: 'invalid_request',
```

Comment:
Are these [error responses](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-3.6) only necessary for the client (not the RO)?

Reply:
It's only for the client. Though looking at the spec, it does seem like these are only necessary during the grant authorization flow and not necessarily for token management endpoints.

Reply:
Yeah, I was thinking that since the RO would be sending token introspection requests that these error responses don't _have_ to be used.

Reply:
Isn't that covered by checking against the spec?